src/WebJobs.Extensions.DurableTask/Options/AzureStorageOptions.cs

```diff
@@ -222,6 +222,11 @@ public void Validate()
             {
                 throw new InvalidOperationException($"{nameof(this.ControlQueueBufferThreshold)} must be between 1 and 1000.");
             }
+
+            if (this.ControlQueueBatchSize > this.ControlQueueBufferThreshold)
+            {
+                throw new InvalidOperationException($"{nameof(this.ControlQueueBatchSize)} cannot be larger than {nameof(this.ControlQueueBufferThreshold)}");
```

Comment:
Not sure if we should explicitly throw an error because that may break customers.

I would say we implicitly change it, or we allow it and log a warning (or change it and log a warning). I don't have a strong preference for which approach we take.

Reply:
Hmm, I'm conflicted here as well. My personal preference is to fail fast because otherwise we can cause confusion, especially if they're trying to do some performance tuning themselves.

That being said, I'm ok with logging a warning and leaving their config as is. How do you feel about including a blurb in the warning about DF, in the next major release, potentially dropping "support" for misconfigurations like this one?

Reply:
In general, fail fast is a good policy for bad configurations, but when they are more awkward than actively harmful, and we have allowed them for a long time, I consider that a breaking change, which I am hesitant to take on, especially in  patch release.

Reply:
Let's go with the log a warning approach.